configure.ac

```diff
@@ -32,7 +32,16 @@ if test "x$enable_libphal" = "xyes"; then
 fi
 
 AC_LANG([C++])
-AX_CXX_COMPILE_STDCXX_17([noext])
+# Adding the required C++ version directly is not typically recommended
```

Comment:
Commit message: 

You could include the test case details that you tried in the commit message, but I'm not sure if the PHAL project follows that practice like the OpenBMC project does.